items.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+
+class ClassName
+  def initialize(id, date, archived: true)
```

Comment:
In constructor, set `archived` as `false`  because when the book will be created, it will not be initially archived 📕